includes/definitions/rutos-2xx.yaml

```diff
@@ -0,0 +1,21 @@
+os: rutos-2xx
+type: network
+group: unix
+text: "Teltonika RutOS"
+icon: teltonika
+ifXmcbc: true
+ifname: true
+mib_dir:
+  - rutos-2xx
```

Comment:
The mib directory should be `teltonika` can you change that and move the mib?

Reply:
done, but there is also MIB for Rut9xx, which I am not using for now, but there is a support for that and it's in a rutos folder, and there is also used TELTONIKA-MIB, but that MIB is different for RUT9XX devices, but same name, so what can we do with that ? create another folder under Teltonika for RUT2XX and another folder for RUT9XX and then move MIBs with a same name here ?

Reply:
yeah, I think subfolders is the best option there.  Silly folks using the same mib name.
The cambium mibs also deal with this.